Packs/Gmail/Integrations/Gmail/Gmail.py

```diff
@@ -456,6 +457,28 @@ def users_to_entry(title, response, next_page_token=None):
     }
 
 
+def labels_to_entry(title, response, user_key):
+    context = []
+
+    for label in response:
+        context.append({
+            'UserID': user_key,
+            'Name': label.get('name'),
+            'ID': label.get('id')
+        })
+    headers = ['Name', 'ID']
+    human_readable = tableToMarkdown(title, context, headers, removeNull=True)
+
+    return {
+        'ContentsFormat': formats['json'],
+        'Type': entryTypes['note'],
+        'Contents': response,
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': human_readable,
+        'EntryContext': {'GmailLabel(val.ID == obj.ID && val.Name == obj.Name && val.UserID == obj.UserID)': context}
+    }
```

Comment:
To use here ```CommandResults``` wouldn't work because it is not wrapped in the main in the ```return_results``` function, right?

Reply:
That's correct. This integration is pretty old and still uses Python2 too. I had to stick with the older method.